hashicorp/boundary-ui
# refactor: 💡 move workers section in a target to be in a tab

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-14332

# Description

Move worker section to its own tab in a target.
Remove oss feature flag `target-worker-filters-v2` and support for deprecated worker_filter field. 




## Screenshots (if appropriate)


https://github.com/hashicorp/boundary-ui/assets/107949262/ef1cca5f-5a87-423c-8b0a-e7eab6d76069


## How to Test


1. open admin ui
2. navigate to a target
3. click on workers tab
4. validate that ingress worker filter is displayed if in ent or hcp editions
5. validate a worker filter can be added and saved

## Checklist
